packages/language-support/src/tests/consoleCommands.test.ts

```diff
@@ -99,7 +107,10 @@ describe('sanity checks', () => {
   });
 
   test('accepts trailing ; ', () => {
-    expectParsedCommands(':history;', [{ type: 'history' }]);
+    expectParsedCommands(':history;', [
+      { type: 'history' },
+      { type: 'parse-error' },
```

Comment:
My gut feeling is that this should not give a parse error here. To me it makes sense that empty string on it's own should be a parse error, but empty string as the last statement in a list should not be considered a statement at all?

Reply:
It should be considered a statement cause I do need the auto-completion to pick up that we are in a new statement to suggest new things.

I've changed this for `{ statement: ''", type: "cypher"}`